central/sensor/service/pipeline/nodes/pipeline.go

```diff
@@ -72,7 +72,7 @@ func (p *pipelineImpl) Match(msg *central.MsgFromSensor) bool {
 	return msg.GetEvent().GetNode() != nil
 }
 
-func (p *pipelineImpl) processRemove(store store.Store, n *storage.Node) error {
+func (p *pipelineImpl) processRemove(store datastore.DataStore, n *storage.Node) error {
```

Comment:
just for consistencies sake, can you change store -> ds

Reply:
will do in postgres pr